projects/dendrites/gaussian_classification/run_dendritic_network.py

```diff
@@ -345,10 +386,12 @@ def evaluate_model(exp):
         for _ in range(num_epochs):
             train_model(exp)
         t2 = time.time()
-        print(f"=== AFTER TASK {task_id} === TRAIN TIME: {t2 - t1}")
 
         # Evaluate model accuracy on each task separately
         if task_id in eval_tasks:
+
```

Comment:
Shouldn't the print statement be before line 391? Right after t2=time.time()?

Reply:
The training time is now only reported for tasks where evaluation is also performed, instead of printing training time for each task. Do you want to have training time for each task reported?

Reply:
I think so - training time is independent of evaluation. You want to see, for example, that the training time doesn't grow with each epoch (can happen with memory issues).

Reply:
Fixed